### PR TITLE
doc: start troubleshoot section

### PR DESCRIPTION
**Pull request by yulaicui**
This PR adds a trouble shooting section to doc that list common errors with corresponding fixes. Reviewers are welcomed to suggest any more issues and fixes.

### CODE REVIEW

**Comment by yulaicui** [comment]
I just created this doc to gather problems and their fixes: [https://docs.google.com/document/d/1K6Jvf66uT3INseqK2qxG67c5q9y0MU_i42FOke_3ItU/edit?usp=sharing]

Feel free to add contents to this document and I'll compile them

**Comment by jiashenC** [comment]
- Computation cannot run on GPUs

File: docs/trouble_shoot/trouble_shoot.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    kill -9 <process_id>
+
+fix #2
+~~~~
+
+Remove eva's running directory. To do this, open a new terminal and run
+
+.. code-block:: bash
+
+    rm -rf .eva
```
Comment (xzdandy) [inline docs/trouble_shoot/trouble_shoot.rst:33]: This is out of date? I think the latest version of eva does not use `.eva`?

**Comment by xzdandy** [comment]
For some issues like `eva_client stuck at a simple query without responding`. I think we should fix it to improve the usability instead of putting them in the trouble shooting section.

File: docs/trouble_shoot/trouble_shoot.rst (hunk 1 above)
Comment (gaurav274) [reply docs/trouble_shoot/trouble_shoot.rst:33]: Yes, we moved to `eva_data`.

**Comment by aryan-rajoria** [comment]
> For some issues like `eva_client stuck at a simple query without responding`. I think we should fix it to improve the usability instead of putting them in the trouble shooting section.

We could

**Comment by jarulraj** [comment]
subsumed in #801

**Pull request closed without merging**